pkg/webhook/controlplane/ensurer.go

```diff
@@ -188,47 +178,19 @@ func ensureKubeAPIServerCommandLineArgs(c *corev1.Container, k8sVersion *semver.
 	}
 }
 
-func ensureKubeControllerManagerCommandLineArgs(c *corev1.Container, k8sVersion *semver.Version) {
+func ensureKubeControllerManagerCommandLineArgs(c *corev1.Container, _ *semver.Version) {
```

Comment:
You could remove the no longer used arg:

```suggestion
func ensureKubeControllerManagerCommandLineArgs(c *corev1.Container) {
```